deploy/olm-catalog/ibm-block-csi-operator/1.0.0/ibm-block-csi-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
-packageName: ibm-block-csi-operator
-channels:
-- name: stable
-  currentCSV: ibm-block-csi-operator.v1.0.0
-defaultChannel: stable
+packageName: ibm-block-csi-operator

```

Comment:
Please don't touch this folder (deploy/olm-catalog/ibm-block-csi-operator) for now, we may need a new folder called 1.1.0 in a separate ticket.

Reply:
Should I also copy-paste `ibm-block-csi-operator.package.yaml` to 1.1.0 directory as is?

Reply:
And should I create it as `ibm-block-csi-operator.v1.1.0.package.yaml` (instead of `ibm-block-csi-operator.package.yaml`), to be aligned with `ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml`?

Reply:
And if so, would it be problematic to rename the existing `ibm-block-csi-operator.package.yaml` to `ibm-block-csi-operator.v1.0.0.package.yaml`?

Reply:
Please keep as it is for now, I will create new ticket for it.

Reply:
@27149chen Could you share the new ticket number here?